docs/pages/get-started/faq.md

```diff
@@ -348,7 +369,7 @@ These network nodes operate in US jurisdiction in compliance with Office of Fore
 
 To explore the software for the nodes that currently form the XMTP network, see the [xmtp-node-go repo](https://github.com/xmtp/xmtp-node-go).
 
-Ephemera is working toward a phased decentralization of the network.
+Ephemera is working toward a phased decentralization of the network. Have questions or feedback regarding decentralization? Post to the [XMTP Community Forums](https://community.xmtp.org/c/development/ideas/54).
```

Comment:
"Working towards a phased approach..." sounds very "not now" to me. I wonder if we should stop referring to the work as phased and just state what we are working on (replication) and discussing (incentives). Wdyt @neekolas @richardhuaaa?

Reply:
That makes sense to me. Maybe "Ephemera is currently in the process of building the replication and incentive mechanisms for a decentralized network." Depending on how we feel we could link to the repo/tracking task, at least until the XIP/blurb is ready

Reply:
Thank you for calling this out, @saulmc!

And I love your proposed language, @richardhuaaa! Is this the tracking task you mentioned? https://github.com/xmtp/xmtpd/issues/118

Reply:
Sample blurb:
This will be comprised of the following components:
- An on-chain smart contract node registry, which lists the currently active set of XMTP nodes
- An off-chain broadcast network of nodes run by independent third parties, for a majority of the messaging data (loosely ordered, low latency)
- A series of on-chain smart contracts on an XMTP L2, for account information and the most critical messaging data (strictly ordered, medium latency)

Reply:
Thank you both for your feedback here, @saulmc and @richardhuaaa! I've updated the content here: https://github.com/xmtp/docs-xmtp-org/pull/26/files#diff-0f5558e2ccc886a61098eac3e4020166d6c4dee87249fc7dcda8eba0f5bd1196R358